sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DisposableDocumentClassifier.cs

```diff
@@ -33,6 +33,11 @@ private DisposableDocumentClassifier(DocumentModelAdministrationClient client, D
         /// </summary>
         public DocumentClassifierDetails Value { get; }
 
+        /// <summary>
+        /// The identifier of the classifier this instance is associated with.
+        /// </summary>
+        public string ClassifierId => Value.ClassifierId;
```

Comment:
Just a convenient property for some tests. We can now call `disposableClassifier.ClassifierId` directly instead of doing `disposableClassifier.Value.ClassifierId`.